beakerx/setup.py

```diff
@@ -101,7 +101,7 @@
         get_data_files(os.path.join('beaker'))
     )],
     install_requires    = [
-        'notebook >=4.4.0',
+        'notebook ==5.6.0',
```

Comment:
Could we get around these tests without locking it to 5.6?

Reply:
yes, we can but I fixed version because I had the following problem
![dockererror](https://user-images.githubusercontent.com/21952713/49997258-64b93400-ff91-11e8-82cb-2b3756fb032a.png)
when I run beakerx docker image

Reply:
thing to consider:
Fixed version of notebook sounds more reliable because when we have "notebook >=4.4.0" we don't really know which version of notebook we have in our docker image and we don't control the  moment when we switch between newer version.
Maybe we should switched to fixed version approach and create notebook version update issue every time when new notebook is released, then we have chance to check if our docker image works fine with newer version of notebook.